department-of-veterans-affairs/abd-vro
# MCP-2545 Swagger pdfTemplate allowableValues

## What was the problem?

Cleanup of swagger specs based on a previous PR review that I was not able to get implemented in time

Associated tickets or Slack threads:

- [MCP-2545](https://amida.atlassian.net/browse/MCP-2545)

## How does this fix it?[^1]


## How to test this PR
- Use Swagger to generate a PDF


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
